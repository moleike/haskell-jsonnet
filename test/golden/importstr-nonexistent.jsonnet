importstr "this file does not exist"
