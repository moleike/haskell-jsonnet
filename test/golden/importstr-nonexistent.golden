Parse error: this file does not exist (No such file or directory)
    ./test/golden/importstr-nonexistent.jsonnet:1:1-37